packages/next-swc/crates/core/src/server_actions.rs

```diff
@@ -465,7 +488,11 @@ impl<C: Comments> VisitMut for ServerActions<C> {
                         ..
                     })) => {
                         let ids: Vec<Id> = collect_idents_in_var_decls(&var.decls);
-                        self.exported_idents.extend(ids);
+                        self.exported_idents.extend(
+                            ids.into_iter()
+                                .map(|id| (id, false))
+                                .collect::<Vec<(Id, bool)>>(),
```

Comment:
`.collect::<Vec<(Id, bool)>>()` seems redundant, as we are passing them to `extend`